src/views/settings/Profile.vue

```diff
@@ -70,16 +114,23 @@ export default class Profile extends Vue {
   /**********************
    * data fields
    **********************/
-
   public user!: InternalActor;
+  public confirmPassword = "";
+  public confirmPwdErrorMsgs: string[] = [];
+  public hasConfirmPwdError = false;
+  public newPassword = "";
+  public password = "";
+  public showConfirmPassword = false;
+  public showNewPassword = false;
+  public showPassword = false;
```

Comment:
To prevent confusion?

```suggestion
  public showCurrentPassword = false;
```